### PR TITLE
Notes for including SSL dependencies

### PR DESCRIPTION
**Pull request by plooploops**
## Summary

These are notes to prevent trimming SSL dependencies with .netcore 3.1 / .net 5.0 apps.

### CODE REVIEW

File: docs/core/deploying/trim-self-contained.md [hunk 1]
```diff
@@ -32,6 +32,39 @@ When the code is indirectly referencing an assembly through reflection, you can
 </ItemGroup>
 ```
 
+### Support for SSL Certificates
+
+Suppose one of the scenarios is to load SSL certificates within an ASP.NET Core app.  We want to ensure that when trimming we also prevent trimming assemblies that will help with loading SSL certificates.
```
Comment (adegeo) [inline docs/core/deploying/trim-self-contained.md:37]: - Remove double space
- I would be more aggressive with the wording. I feel like "Suppose" is too related to suggesting they may want to do something when the it's more like problem a exists here is a solution.

```suggestion
If your app loads SSL certificates, such as in an ASP.NET Core app, you'll want to ensure that when trimming you prevent trimming assemblies that will help with loading SSL certificates.
```

File: docs/core/deploying/trim-self-contained.md [hunk 2]
```diff
@@ -32,6 +32,39 @@ When the code is indirectly referencing an assembly through reflection, you can
 </ItemGroup>
 ```
 
+### Support for SSL Certificates
```
Comment (adegeo) [inline docs/core/deploying/trim-self-contained.md:35]: - Normal casing for titles.

```suggestion
### Support for SSL certificates
```

**Review by adegeo — APPROVED** [review]
I left some comments for you to consider.

File: docs/core/deploying/trim-self-contained.md (hunk 2 above)
Comment (plooploops) [reply docs/core/deploying/trim-self-contained.md]: Adding in the suggestion, thank you!

File: docs/core/deploying/trim-self-contained.md (hunk 1 above)
Comment (plooploops) [reply docs/core/deploying/trim-self-contained.md]: Adding in the suggestion, thank you!

**Comment by plooploops** [comment]
@adegeo thank you for the approval.  Let me know if there's other checks we need to do before merging.

**Pull request merged**